src/main/webapp/EditHelp.js

```diff
@@ -0,0 +1,100 @@
+                $(thistr).insert({after:customtr});
+                $(customtr).down().next().down().down().down().next().style.display = "block";
+                var div = $(tr).down().next().down();
+                var div2 = $(customtr).down().next().down().down().next()
+                var div3 = $(customtr).down().next().down();
+                div.style.display = "block";
+                div3.style.display = "block";
+
+                // make it visible
+                new Ajax.Request(this.getAttribute("helpURL"), {
+                    method : 'get',
+                    onSuccess : function(x) {
+                        var from = x.getResponseHeader("X-Plugin-From");
+                        div.innerHTML = x.responseText+(from?"<div class='from-plugin'>"+from+"</div>":"");
+                        layoutUpdateCallback.call();
+                    },
+                    onFailure : function(x) {
+                        div.innerHTML = "<b>ERROR</b>: Failed to load help file: " + x.statusText;
+                        layoutUpdateCallback.call();
+                    }
+                });
+                
+                var customHelpUrl = this.getAttribute("helpURL").replace(rootURL+"/", "").replace("/", ".");
+                while(customHelpUrl.indexOf("/") + 1){
+                    customHelpUrl = customHelpUrl.replace("/", ".")
+                }
+
+                var customurl = rootURL + "/helpmanager/get?class="+customHelpUrl;
+                    $(div2).setAttribute("customHelpUrl", rootURL+"/helpmanager/update");
+                    $(div2).setAttribute("className", customHelpUrl);
+
+                new Ajax.Request(customurl, {
+                    method : 'get',
+                    onSuccess : function(x) {
+                        var from = x.getResponseHeader("X-Plugin-From");
+                        div2.innerHTML = x.responseText+(from?"<div class='from-plugin'>"+from+"</div>":"");
+                        layoutUpdateCallback.call();
+                    },
+                    onFailure : function(x) {
+                        div2.innerHTML = "<b>ERROR</b>: Failed to load help file: " + x.statusText;
+                        layoutUpdateCallback.call();
+                    }
+                });
+
+                } else {
+                    var div = $(findFollowingTR(tr, "help-area")).down().next().down();
+                    div.style.display = "none";
+                    $(tr).remove();
+                    layoutUpdateCallback.call();
+                }
+
+            return false;
+        };
+
+        e.tabIndex = 9999; 
+        e = null;
+        
+    });
+
+    function save_custom_help(myel) {
+        $(myel).style.display = "none";
+        $(myel).next().style.display = "block";
+        var div = $(myel).up().next();
+        div.innerHTML = div.down().value;
+
+        var xmlhttp;
+        if (window.XMLHttpRequest) {
```

Comment:
@daniel-beck I have difficulties with ajax request because it seems like Stapler does not see data sent by POST request on the update page when ajax request is used.